docs/TESTS.md

```diff
@@ -28,8 +34,8 @@ Optionally, Xcode can create a third file for experimentation:
 
 
 ### Configuring Xcode
-1. In the Terminal `cd` to the directory for an exercise (for example, ~/exercism/swift/exercises/hello-world/), then execute `swift package generate-xcodeproj`. This creates an Xcode project in the current directory with the same name as the package. (The generated project is ideal for unit tests. It does not include the overhead of launching the iOS simulator or a target application.)
-2. Open the newly generated project file.
+1. In the Terminal `cd` to the directory for an exercise (for example, ~/exercism/swift/exercises/hello-world/), then type `swift package generate-xcodeproj` and press enter. This creates an Xcode project in the current directory with the same name as the package. When you run this command, Terminal prints out a statement showing you the name of the xcode project it has created. For the HelloWorld exercise, you will see: `generated: ./HelloWorld.xcodeproj`. (The generated project is ideal for unit tests. It does not include the overhead of launching the iOS simulator or a target application.)
```

Comment:
By the way, we can get away with just `cd` + `xed .`

As for me there's no need to call `swift package generate-xcodeproj`. XCode can do that automatically.

`xed` ships with XCode. [More about xed](https://medium.com/@dive/xcode-invocation-tool-xed-deb04d03cf98).